GitCommands/Settings/AppSettings.cs

```diff
@@ -686,6 +686,12 @@ public static bool RebaseAutoStash
             set => SetBool("RebaseAutostash", value);
         }
 
+        public static bool SkipRebaseDialog
+        {
+            get { return GetBool("SkipRebaseDialog", true); }
```

Comment:
We have to decide either it is an advanced option and should default to `false` or it defaults to `true` and we place it somewhere outside of the advanced node.

Reply:
I vote for false and keep it in advanced

The third decision is is if there should be a popup if not set (as for update submodules. I vote no.

Reply:
agree